src/OrbitQt/HistogramWidgetTest.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <algorithm>
+
+#include "HistogramWidget.h"
+
+namespace orbit_qt {
+static void TestGetBinWidths(int number_of_bins, int histogram_width) {
+  const std::vector<int> widths = GenerateBinWidth(number_of_bins, histogram_width);
+  ASSERT_EQ(widths.size(), number_of_bins);
+  int sum = std::reduce(std::begin(widths), std::end(widths), 0);
+  ASSERT_EQ(sum, histogram_width);
+  int max = *std::max_element(std::begin(widths), std::end(widths));
+  int min = *std::min_element(std::begin(widths), std::end(widths));
+  ASSERT_LE(max - min, 1);
+  ASSERT_THAT(widths, testing::Each(testing::Ge(0)));
+}
+
+TEST(HistogramUtil, GetBinWidthIsCorrect) {
```

Comment:
Rename this to match the name of the function. Also change `HistogramUtil` to `HistrogramWidget`.